lib/fishjam/room/config.ex

```diff
@@ -30,7 +34,11 @@ defmodule Fishjam.Room.Config do
 
   @spec from_params(map()) :: {:ok, __MODULE__.t()} | {:error, atom()}
   def from_params(params) do
-    room_id = Map.get(params, "roomId")
+    room_id =
+      if FeatureFlags.custom_room_name_disabled?(),
+        do: ID.generate(),
+        else: Map.get(params, "roomId")
```

Comment:
I think that logic regarding `room_id` should be in the same place; currently, we separately have this, and separately, we have to parse room_id.